loopchain/channel/channel_inner_service.py

```diff
@@ -241,16 +240,14 @@ def get_tx_info(self, tx_hash):
 
         self._channel_service.state_machine.vote()
 
-        if unconfirmed_block.made_block_count >= conf.LEADER_BLOCK_CREATION_LIMIT \
-                and unconfirmed_block.block_type is BlockType.vote \
-                and unconfirmed_block.is_divided_block is False:
+        if unconfirmed_block.block_type is BlockType.vote and unconfirmed_block.is_divided_block is False:
```

Comment:
```suggestion
        if unconfirmed_block.block_type is BlockType.vote and unconfirmed_block.is_divided_block:
```
if is_divided_block is bool value why compare with bool?

Reply:
This is not a modification in this PR.